ui/pom.xml

```diff
@@ -15,7 +15,7 @@
         <junit.version>4.13.2</junit.version>
         <aspectj.version>1.9.8.M1</aspectj.version>
         <assertj.version>3.20.2</assertj.version>
-        <cucumber.version>6.0.0</cucumber.version>
+        <cucumber.version>7.0.0-RC1</cucumber.version>
```

Comment:
please don't use artefact with RC postfix, RC means release candidate - it's not production ready

Reply:
noted and corrected